vite.config.production.ts

```diff
@@ -51,7 +51,8 @@ export const config: UserConfigFn = ({ mode }) => {
                 // for which we use FormatJS under the hood. FormatJS requires you to escape any XML/HTML looking
                 // things, plus ICU '{' and '}', hence this replace.
                 // If we ever need !!text/markdown for anything else we should do something like !!text/icu+markdown
-                return marked(data).replace(/</g, "'<'")
+                const str = marked(data) as string
```

Comment:
**Comment**: Awaiting this didn’t work and made a bunch of tests fail. That’s not super surprising given that this a callback function passed to another thing to consume.

It looks like in our use, we’re not actually getting a promise, so type assertion should be fine here.